.github/workflows/ci.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: CI
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+env:
+  RAILS_ENV: test
+  DATABASE_ADAPTER: sqlite3
+  SCMS: git
+
+concurrency:
+  group: ${{ github.ref_name }}
+  cancel-in-progress: true
+
+jobs:
+  ruby:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - '3.0'
+          - '3.1'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: true
+      - name: Install dependencies
+        run: sudo apt-get install -y gsfonts
```

Comment:
It surprises me you don't need to run update too.

Reply:
it runs on the GHA "thing", which does have a populated cache (vs a "real" container that has no apt cache)